src/main/java/bio/terra/service/dataset/flight/create/DatasetCreateFlight.java

```diff
@@ -33,26 +36,45 @@ public DatasetCreateFlight(FlightMap inputParameters, Object applicationContext)
         IamProviderInterface iamClient = (IamProviderInterface) appContext.getBean("iamProvider");
         ConfigurationService configService = (ConfigurationService) appContext.getBean("configurationService");
         ProfileService profileService = (ProfileService) appContext.getBean("profileService");
+        AzureDataLocationSelector azureDataLocationSelector =
+            (AzureDataLocationSelector) appContext.getBean("azureDataLocationSelector");
+        DatasetStorageAccountDao datasetStorageAccountDao =
+            (DatasetStorageAccountDao) appContext.getBean("datasetStorageAccountDao");
```

Comment:
This is a pattern in our code we can improve. I believe these beans don't need to be looked up by name; if so then the code could use `getBean(Class.class)`. If we do need to look up by name, we can at least use the API that also accepts a class argument.
```suggestion
        AzureDataLocationSelector azureDataLocationSelector =
            appContext.getBean("azureDataLocationSelector", AzureDataLocationSelector.class);
        DatasetStorageAccountDao datasetStorageAccountDao =
            appContext.getBean("datasetStorageAccountDao", DatasetStorageAccountDao.class);
```

Reply:
Yeah, I've been wanting to do a wholesale change over to the class instantiation